cmd/daytona/config/const.go

```diff
@@ -113,6 +116,8 @@ func GetRequiredScopesFromGitProviderId(providerId string) string {
 		return "Code (Status, Read & Write); User Profile (Read); Project and Team (Read)"
 	case "aws-codecommit":
 		return "/"
+	case "gogs":
+		fallthrough
```

Comment:
Gogs does not allow fine-grained tokens. Each token will have full access to your account.

Reply:
![Screenshot 2024-10-16 at 13 26 16](https://github.com/user-attachments/assets/9a2eb08f-b2b0-4543-8a10-cb2af97346aa)

We should hide the `Required Scopes:` label if it's empty

Reply:
done